starters/graphql-serverless-contentful/src/schema/post/post.resolver.test.ts

```diff
@@ -0,0 +1,204 @@
+			sys: {
+				id: MOCK_ID,
+			},
+			fields: {
+				content: {
+					'en-US': content,
+				},
+			},
+		})),
+	};
+
+	const MOCK_CONTENTFUL_ENVIRONMENT = {
+		getEntry: jest.fn().mockReturnValue({
+			...MOCK_CONTENTFUL_ENTRIES.items[0],
+			publish: jest.fn().mockReturnValue({}),
+			update: jest.fn().mockReturnValue({}),
+			updatePost: {
+				id: MOCK_ID,
+				content: MOCK_CONTENT[1],
+			},
+		}),
+		createEntry: jest.fn().mockReturnValue({
+			...MOCK_CONTENTFUL_ENTRIES.items[0],
+			publish: jest.fn().mockReturnValue({
+				createPost: {
+					id: MOCK_ID,
+					content: MOCK_CONTENT[0],
+				},
+			}),
+		}),
+		getEntries: jest.fn().mockReturnValue(MOCK_CONTENTFUL_ENTRIES),
+	};
+	const MOCK_CONTENTFUL_SPACE = {
+		getEnvironment: jest.fn().mockReturnValue(MOCK_CONTENTFUL_ENVIRONMENT),
+	};
+	const MOCK_CONTENTFUL_CLIENT = {
+		client: {
+			getSpace: jest.fn().mockReturnValue(MOCK_CONTENTFUL_SPACE),
+		},
+	};
+	return MOCK_CONTENTFUL_CLIENT;
+});
+
+describe('post queries and mutations', () => {
+	let post: any;
+	let content: string;
+
+	beforeAll(async () => {
+		content = 'This is the test post content';
+		const MUTATION = gql`
+			mutation CreatePostMutation($content: String!) {
+				createPost(content: $content) {
+					id
+					content
+				}
+			}
+		`;
+
+		const res: GraphQLResponse = await apolloServer.executeOperation({
+			query: MUTATION,
+			variables: {
+				content,
+			},
+		});
+
+		assert(res.body.kind === 'single');
+		expect(res.body.singleResult.errors).toBeUndefined();
+		assert(res.body.singleResult.data);
+
+		post = res.body.singleResult.data.createPost;
+	});
+
+	describe('query posts', () => {
+		it('returns created post', async () => {
+			const QUERY = gql`
+				query {
+					posts {
+						id
+						content
+					}
+				}
+			`;
+
+			const subject: GraphQLResponse = await apolloServer.executeOperation({
+				query: QUERY,
+			});
+			assert(subject.body.kind === 'single');
+			expect(subject.body.singleResult.errors).toBeUndefined();
+			assert(subject.body.singleResult.data);
+
+			const createdPosts = subject.body.singleResult.data.posts;
+			expect(createdPosts).toContainEqual({
+				id: post.id,
+				content: post.content,
+			});
+		});
+	});
+
+	describe('query post by id', () => {
+		it('returns the post with the given id', async () => {
+			const QUERY = gql`
+				query PostQuery($id: ID!) {
+					posts(id: $id) {
+						id
+						content
+					}
+				}
+			`;
+
+			const subject: GraphQLResponse = await apolloServer.executeOperation({
+				query: QUERY,
+				variables: {
+					id: post.id,
+				},
+			});
+			assert(subject.body.kind === 'single');
+			expect(subject.body.singleResult.errors).toBeUndefined();
+			assert(subject.body.singleResult.data);
+			expect(subject.body.singleResult.data).toEqual({
+				posts: [
+					{
+						id: post.id,
+						content: post.content,
+					},
+				],
+			});
+		});
+	});
+
+	describe('mutation createPost', () => {
+		let created_post: any;
```

Comment:
This definition can move to line 170 and be turned into a `const`
But also see my other comment of asserting `subject.body.singleResult.data.createPost` directly